pkg/controllers/disruption/consolidation_test.go

```diff
@@ -214,6 +217,87 @@ var _ = Describe("Consolidation", func() {
 				metrics.ReasonLabel: "underutilized",
 			})
 		})
+		DescribeTable("should correctly report invalidated commands for emptiness disruption when", func(invalidatedReason string, validatorOpt TestEmptinessValidatorOption) {
```

Comment:
When you say "correctly report invalidated commands", does that mean that we return back modified commands?

Reply:
No just the metrics for them. I can make clarifications to the suite.